packages/rocketchat-livechat/client/views/app/livechatAnalytics.html

```diff
@@ -0,0 +1,56 @@
+<template name="livechatAnalytics">
```

Comment:
What do you think of creating a new folder to store the analytics client files?
He already do this to store `triggers` client files in `Rocket.Chat/packages/rocketchat-livechat/client/views/app/triggers`.